infrastructure/environments/dplplat01/configuration/harbor/harbor-values.template.yaml

```diff
@@ -61,3 +61,20 @@ persistence:
       accountname: "$STORAGE_ACCOUNT_NAME"
       accountkey: "$STORAGE_ACCOUNT_KEY"
       container: "$STORAGE_CONTAINER_NAME"
+
+# Use tolerations
+tolerations:
+  - key: noderole.dplplatform
+    effect: NoSchedule
+    operator: Equal
+    value: system
```

Comment:
Its the admin node - also you need to fix the effect.

Reply:
oh right